loadtest/set_workers.sh

```
@@ -0,0 +1,86 @@
+# a previously patched copy, so worker counts can't stack up. loadtest/pm.sh is
+# deliberately NOT used: it hardcodes 4 workers and was never the file Play ran.
+#
+# process_monitor.sh is the container entrypoint, so changing it needs a
+# `docker restart`. That is fine BETWEEN configs; never do it mid-trial.
+# `docker restart` preserves docker cp'd files, so the phase-timing patch
+# survives -- only `docker compose up -d` would wipe it.
+set -euo pipefail
+
+N=${1:?usage: set_workers.sh <N|restore>}
+CONTAINER=${CONTAINER:-${PIPESHUB_CONTAINER:-pipeshub-ai}}
+. "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/_common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./_common.sh was not specified as input (see shellcheck -x).

(SC1091)